### PR TITLE
Update IEP to match designs and handle errors/partial data

### PR DESCRIPTION
**Pull request by LucasCairns**

### CODE REVIEW

File: assets/sass/components/_personalised.scss [hunk 1]
```diff
@@ -85,6 +81,11 @@
   background-color: govuk-colour('purple');
   margin-right: 0;
 }
+.iep-level--placeholder {
+  color: govuk-colour('black');
+  background-color: rgba(177, 180, 182, 0.5);
```
Comment (psoleckimoj) [inline assets/sass/components/_personalised.scss:86]: Does this need opacity or is it meant to be a solid colour?

Comment (LucasCairns) [reply assets/sass/components/_personalised.scss:86]: Just to be consistent with the balance placeholders IIRC

Comment (psoleckimoj) [reply assets/sass/components/_personalised.scss:86]: OK 👍

**Review by psoleckimoj — APPROVED** [review]
LGTM

**Pull request merged**